<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KindOfDevLogForm</class>
 <widget class="QWidget" name="KindOfDevLogForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>932</width>
    <height>424</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>932</width>
    <height>424</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>932</width>
    <height>424</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>KindOfDevLogForm</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="Line" name="line_7">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_numLabel">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>设备编号:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_numLineEdit">
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>25</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>5</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_typeLabel">
       <property name="maximumSize">
        <size>
         <width>95</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>巡检时状态:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="m_comboBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <item>
        <property name="text">
         <string>任意状态</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>正常运转</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>风险等级Ⅰ</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>风险等级Ⅱ</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>风险等级Ⅲ</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>故障停工</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>5</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="Line" name="line_6">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_findButton">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>点击查询</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../Resource.qrc">
         <normaloff>:/res/tab/find.ico</normaloff>:/res/tab/find.ico</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_clearButton">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>点击清除</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../Resource.qrc">
         <normaloff>:/res/tab/clear.ico</normaloff>:/res/tab/clear.ico</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_8">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTableWidget" name="m_devTable">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>150</number>
       </attribute>
       <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderShowSortIndicator" stdset="0">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderStretchLastSection">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>设备编号</string>
        </property>
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="textAlignment">
         <set>AlignLeading|AlignTop</set>
        </property>
        <property name="icon">
         <iconset resource="../../../Resource.qrc">
          <normaloff>:/res/tab/id.ico</normaloff>:/res/tab/id.ico</iconset>
        </property>
       </column>
       <column>
        <property name="text">
         <string>设备类型</string>
        </property>
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="textAlignment">
         <set>AlignLeading|AlignTop</set>
        </property>
        <property name="icon">
         <iconset resource="../../../Resource.qrc">
          <normaloff>:/res/tab/kind.ico</normaloff>:/res/tab/kind.ico</iconset>
        </property>
       </column>
       <column>
        <property name="text">
         <string>巡检时状态</string>
        </property>
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="textAlignment">
         <set>AlignLeading|AlignTop</set>
        </property>
        <property name="icon">
         <iconset resource="../../../Resource.qrc">
          <normaloff>:/res/tab/statu.ico</normaloff>:/res/tab/statu.ico</iconset>
        </property>
       </column>
       <column>
        <property name="text">
         <string>巡检时间</string>
        </property>
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="textAlignment">
         <set>AlignLeading|AlignTop</set>
        </property>
        <property name="icon">
         <iconset resource="../../../Resource.qrc">
          <normaloff>:/res/tab/time.ico</normaloff>:/res/tab/time.ico</iconset>
        </property>
       </column>
       <column>
        <property name="text">
         <string>库ID</string>
        </property>
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="icon">
         <iconset resource="../../../Resource.qrc">
          <normaloff>:/res/logo/exelogo.ico</normaloff>:/res/logo/exelogo.ico</iconset>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_msgLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>ForbiddenCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../Resource.qrc"/>
 </resources>
 <connections/>
</ui>
